UserInterface/ViewModels/MainWindowViewModel.cs

```diff
@@ -153,4 +165,17 @@ private void PreviousViewModelChanged()
     /// This method is only used to make sure the application can't be stopped using alt+f4
     /// </summary>
     private void DoNothing() { }
+
+    private void GetColorSetting()
+    {
+        Settings savedSettings = settingService.GetSetting();
+        System.Windows.Media.Color color = (System.Windows.Media.Color)System.Windows.Media.ColorConverter.ConvertFromString(savedSettings.Color);
```

Comment:
Kan dit niet boven gedefined worden? (System.Windows.Media.ColorConverter.ConvertFromString)?

Reply:
of `using System.Windows.Media;`

Reply:
dat is wat ik bedoel :))

Reply:
Using system.windows.media heb ik er ook in staan alleen was dat niet voldoende om het verschil te kunnen zien tussen System.Drawing.Color, Systen.Windows.Media.Color en BusinessLogic.Classes.Color